skymapper/map.py

```diff
@@ -390,6 +392,8 @@ def grid(self, sep=30, parallel_fmt=None, meridian_fmt=None, lat_min=-90, lat_ma
             lon_max: maximum longitude for graticules
             **kwargs: styling of `matplotlib.lines.Line2D` for the graticules
         """
+        if sep_lat is None or sep_lon is None:
+            sep_lat = sep_lon = sep
```

Comment:
I like the functionality! But this implementation is problematic. If you set `set=30`, and `sep_lon=None`, then `sep_lat` will be overwritten to `sep` whether it was set or not.

I recommend doing this differently. Let's make `sep` either a number or a list of 2 numbers, which will the be unpacked into `sep_lat` and `sep_lon`.

Reply:
Sure! I've implemented it that way instead - it should allow for any two-element list/tuple/array, etc.